sos/component.py

```diff
@@ -169,12 +204,13 @@ def load_options(self):
             if option.default != SUPPRESS:
                 option.default = None
 
-        # load values from cmdline
-        cmdopts = SoSOptions().from_args(self.parser.parse_args(self.cmdline))
-        opts.merge(cmdopts)
+        opts.update_from_conf(self.args.config_file, self.args.component)
+        if os.getuid() != 0:
```

Comment:
Why `root` cant use its own config file as well? I am quite OK with this, just dont see much rationale (can't root require different defaults than system ones?).

Reply:
I don't mind removing the restriction, but my rationale here was simply that it's not a common trend.